src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundAssignmentTests.cs

```diff
@@ -1091,6 +1091,27 @@ void M()
         {
         }
     }
+}");
+        }
+
+        [WorkItem(38054, "https://github.com/dotnet/roslyn/issues/53969")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseCompoundAssignment)]
+        public async Task TestIncrementInExpressionContext()
+        {
+            await TestInRegularAndScript1Async(
+@"public class C
+{
+    void M(int i)
+    {
+        M(i [||]= i + 1);
+    }
+}",
+@"public class C
+{
+    void M(int i)
+    {
+        M(++i);
```

Comment:
Without a call to `WithoutTrailingTrivia()`, the fix produces `M(++i );`

Reply:
the whole new expression should probably hve teh the exact leading/trailing trivia as the original full expression.  WithTriviaFrom can help with that.

Reply:
Done in 9e428fe7